pilot/pkg/networking/grpcgen/grpcgen_test.go

```diff
@@ -96,14 +102,28 @@ func TestGRPC(t *testing.T) {
 		},
 	})
 
+	store.Create(config.Config{
```

Comment:
confused -- why was this not required before but is now?

Reply:
This will make sure we build TLS context in CDS, now the test covers whether gRPC NACKs that config.